packages/forms-web-app/src/pages/projects/documents/view.njk

```diff
@@ -56,51 +57,62 @@
 
 				{% if documents.length != 0 or queryUrl != '' %}
 					<h2 class="govuk-heading-m">
-						Search documents
+						{{ t('projectsDocuments.heading2') }}
 					</h2>
 
 					<p class="govuk-body">
-						Search by author, description or document type.
+						{{ t('projectsDocuments.paragraph2') }}
 					</p>
 
-					{{ uiSearchBar('documents-page-search-bar', 'searchTerm', searchTerm, 'Search documents') }}
+					{{ uiSearchBar('documents-page-search-bar', 'searchTerm', searchTerm, 'Search documents', t('common.search') ) }}
 				{% endif %}
 
 				{{ uiTagLinkList('Filtered results', activeFilters) }}
 
 				{% if documents.length > 0 %}
 					{% if displayClearAllFilters %}
-						<p>
+						<p class="govuk-body">
 							<a class="govuk-link" href="{{ baseUrl }}/{{ pageUrl }}">
-								Clear all filters
+								{{ t('projectsDocuments.linkText1') }}
 							</a>
 						</p>
 					{% endif %}
 
-					<p>
-						Showing <strong>{{ paginationData.fromRange }}</strong> to <strong>{{ paginationData.toRange }}</strong> of
-						<strong>{{ paginationData.totalItems }}</strong> documents, newest first.
+					<p class="govuk-body">
+						{{ t('projectsDocuments.paragraph3', { from: '<strong>' + paginationData.fromRange + '</strong>', to: '<strong>' + paginationData.toRange + '</strong>', total: '<strong>' + paginationData.totalItems + '</strong>' }) | safe }}
```

Comment:
Will from: , to: and total: be converted to welsh.

Reply:
_We are still waiting on a number of translations that are missing, these are noted in the comments section on the ticket. I will hold off merging until these can be updated._